addons/mrp/models/mrp_production.py

```diff
@@ -1883,6 +1883,62 @@ def action_serial_mass_produce_wizard(self):
         }
         return action
 
+    def action_split_wizard(self):
+        if len(self) > 1:
+            if any(production.state not in ('draft', 'confirmed') for production in self):
+                raise ValidationError(_("Some manufacturing orders cannot be split."))
+            productions = [Command.create({'production_id': production.id}) for production in self]
+            wizard = self.env['mrp.production.split'].create({'production_ids': productions})
+            action = self.env['ir.actions.actions']._for_xml_id('mrp.action_mrp_production_split')
+            action['res_id'] = wizard.id
+            return action
+        else:
+            if self.state not in ('draft', 'confirmed'):
+                raise ValidationError(_("This manufacturing order cannot be split."))
+            wizard = self.env['mrp.production.split.production'].create({'production_id': self.id})
+            action = self.env['ir.actions.actions']._for_xml_id('mrp.action_mrp_production_split_production')
+            action['res_id'] = wizard.id
+            return action
+
+    def action_merge(self):
+        if len(self) == 1:
+            raise UserError(_("Nothing to Merge."))
+        if any(production.state not in ('draft', 'confirmed') for production in self):
+            raise ValidationError(_("Some manufacturing orders cannot be merged."))
+        products = set([(production.product_id, production.bom_id) for production in self])
+        if len(products) > 1:
+            raise UserError(_('You can only merge manufacturing orders of identical products.'))
+        product_id, bom_id = products.pop()
+        users = set([production.user_id for production in self])
+        if len(users) == 1:
+            user_id = users.pop()
+        else:
+            user_id = self.env.user
+
+        self.action_cancel()
+
```

Comment:
We should also unlink them no? It's a bit too soft for move linked to child move. We will lose the links.
Maybe we should prevent the MTO to be merge if a custom attribute is set on the stock.move